docs/docs/extend/plugins/settings.md

```diff
@@ -52,6 +52,11 @@ class PluginWithSettings(SettingsMixin, InvenTreePlugin):
                 MaxValueValidator(25)
             ]
         },
+        'HIDDEN_SETTING': {
+            'name': _('Hidden Setting'),
+            'description': _('This setting is hidden from the automatically generated plugin settings page'),
+            'hidden': True,
+        }
```

Comment:
@SchrodingersGat That's why I added this. You want me to add a small note somewhere else?

Reply:
Yeah a small callout near that would be good

```markdown
!!! tip "Hidden Settings"
    Plugin settings can be hidden from the settings page by marking them as 'hidden'
```

Reply:
Added it. Do you know suggestions code blocks? That way we could just click on apply directly on GitHub.

Reply:
didn't know that, I'll keep it in mind :)